mvp.feature

```diff
@@ -21,13 +22,15 @@ Feature: MVP features for NVA
     And they see their name in the Menu
     And they see the Connect Author dialog
 
+  @done
```

Comment:
Can a task be done and work-in-progress ?

Reply:
I think the `@wip` was there from before. It should probably be removed